packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -154,6 +154,7 @@ class FloatingActionButton extends StatelessWidget {
        assert(highlightElevation == null || highlightElevation >= 0.0),
        assert(disabledElevation == null || disabledElevation >= 0.0),
        assert(mini != null),
+       assert(clipBehavior != null),
```

Comment:
Breaking

Reply:
No, this would have failed in MaterialButton.